<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QDialog" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>69</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="sex">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>129</y>
     <width>54</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="birthPlace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>209</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="livingPlace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>69</y>
     <width>26</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>100</y>
     <width>26</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>131</y>
     <width>26</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>167</y>
     <width>52</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>209</y>
     <width>39</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>出生地</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>240</y>
     <width>39</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>居住地</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthDate">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>167</y>
     <width>118</width>
     <height>32</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1996</year>
     <month>8</month>
     <day>27</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>72</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Kaiti TC</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户添加</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddUser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
